packages/daemon/src/app/controller/job.ts

```diff
@@ -82,9 +82,6 @@ export class JobController {
     const { pipelineId, offset, limit } = ctx.query;
     try {
       const jobs = await this.pipelineService.queryJobs({ pipelineId }, { offset, limit });
-      if (!jobs || jobs.count === 0) {
-        throw new TypeError('job not found');
-      }
```

Comment:
Why removing this check?

Reply:
list jobs means "I want check the job list", it should not be an error for API.

Reply:
Reasonable!